components/automate-ui/e2e/deprecated-routes.e2e-spec.ts

```diff
@@ -13,7 +13,6 @@ describe('Deprecated routes redirect to the correct new route for backwards comp
     ['/admin/tokens/my-object',                '/settings/tokens/my-object'],
     ['/admin/teams',                           '/settings/teams'],
     ['/admin/teams/my-object',                 '/settings/teams/my-object'],
-    ['/admin/add-team',                        '/settings/add-team'],
```

Comment:
It would be nice if our create modals were in a route fragment like `/teams#create-team` similar as we do for tabs in the details pages. Navigating to that route would go to the list page and open the create modal. Then we could route this deprecated route to that and not break backwards compatibility.

Reply:
We're going to leave this for now, on the assumption there aren't that many (if any) customers that have bookmarked these old routes and we'd rather stay consistent with the other list pages, which don't use fragments. (though that could change)

Reply:
How come we don't think customers have this specific route bookmarked? I'm fine with it but in general let's not break backwards compatibility. Would be nice to add the fragment functionality regardless but there are a bunch of places to do that so I'll follow up with @susanev if that's behavior we want and we can make a card.